src/components/Header/header.jsx

```diff
@@ -1,17 +1,35 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { Logo } from '../Logo';
 import { Search } from '../Search';
+import { Modal } from '../Modal';
+import { AddForm } from '../Forms/addForm';
 
 import './header.scss';
 
-export const Header = () => (
-  <header className="header">
-    <div className="wrapper">
-      <div className="header__container">
-        <Logo />
-        <button className="header__add-button" type="button">+ add movie</button>
+export const Header = () => {
+  const [isAddModalOpen, setIsAddModalOpen] = useState(false);
+
+  return (
+    <header className="header">
+      <div className="wrapper">
+        <div className="header__container">
+          <Logo />
+          <button
+            className="header__add-button"
+            type="button"
+            onClick={() => setIsAddModalOpen(true)}
+          >
+            + add movie
+          </button>
+        </div>
+        <Search />
       </div>
-      <Search />
-    </div>
-  </header>
-);
+
+      {isAddModalOpen && (
+      <Modal setIsModalOpen={setIsAddModalOpen}>
```

Comment:
Not an error, but i think that there is more sense to pass function to close modal, like this:
```js
closeModal={() => setIsAddModalOpen(false)}
```

Reply:
Fixed, thanks